### PR TITLE
Fix Ctrl+H layer hiding and Ctrl+L layer locking only working with the graph open

### PR DESCRIPTION
**Pull request by faisalill**
## CLOSES
image

### CODE REVIEW

**Comment by Keavon** [comment]
Nice! Can you please also extend this to Ctrl+L to toggle locking? Should be a pretty quick change that does the same as you did with Ctrl+H.

**Comment by faisalill** [comment]
@Keavon DONE

File: editor/src/messages/input_mapper/input_mappings.rs [hunk 1]
```diff
@@ -311,6 +309,8 @@ pub fn input_mappings() -> Mapping {
 		entry!(KeyDown(KeyP); modifiers=[Alt], action_dispatch=DocumentMessage::DebugPrintDocument),
 		entry!(KeyDown(KeyO); modifiers=[Alt], action_dispatch=DocumentMessage::ToggleOverlaysVisibility),
 		entry!(KeyDown(KeyS); modifiers=[Alt], action_dispatch=DocumentMessage::ToggleSnapping),
+		entry!(KeyDown(KeyH); modifiers=[Control], action_dispatch=DocumentMessage::ToggleSelectedVisibility),
+		entry!(KeyDown(KeyL); modifiers=[Control], action_dispatch=DocumentMessage::ToggleSelectedLocked),
```
Comment (Keavon) [inline editor/src/messages/input_mapper/input_mappings.rs:313]:
```suggestion
		entry!(KeyDown(KeyH); modifiers=[Accel], action_dispatch=DocumentMessage::ToggleSelectedVisibility),
		entry!(KeyDown(KeyL); modifiers=[Accel], action_dispatch=DocumentMessage::ToggleSelectedLocked),
```

**Review by Keavon — APPROVED** [review]

**Comment by Keavon** [comment]
!build

**Comment by Keavon** [comment]
Thanks for the help!

**Pull request merged**